packages/tools/devtools/devtools-view/src/components/TelemetryView.tsx

```diff
@@ -70,7 +57,7 @@ export function TelemetryView(): React.ReactElement {
 	 */
 	const [bufferedEvents, setBufferedEvents] = React.useState<ITimestampedTelemetryEvent[]>([]);
 	const [maxEventsToDisplay, setMaxEventsToDisplay] = React.useState<number>(DEFAULT_PAGE_SIZE);
-	const [selectedIndex, setSelectedIndex] = React.useState<number | undefined>();
+	const [selectedIndex, setSelectedIndex] = React.useState<number>(0);
```

Comment:
Doesn't this set our selection automatically to the first item in the list? Don't we want to start with no selection at all?